examples/README.md

```diff
@@ -106,30 +106,30 @@ When you open a notebook, select the kernel `nvflare_example` using the dropdown
-| [Federated Statistics for DataFrame](./advanced/federated-statistics/df_stats/README.md)                                               | -            | [Data Frame Federated Statistics](./advanced/federated-statistics/df_stats.ipynb), [Visualization](./advanced/federated-statistics/df_stats/demo/visualization.ipynb)  | see requirements.txt   | Example of gathering local statistics summary from Pandas DataFrame to compute the global dataset statistics.                                                   |
+| Example                                                                                                                                |  Notebooks                                                                                                                                                             | Notes                  | Summary                                                                                                                                                         |
+|----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Federated Statistics Overview](./advanced/federated-statistics/README.md)                                                             | -                                                                                                                                                                      | -                      | Discuss the overall federated statistics features.                                                                                                              |
+| [Federated Statistics for Medical Imaging](./advanced/federated-statistics/image_stats/README.md)                                      | [Image Histograms](./advanced/federated-statistics/image_stats.ipynb)                                                                                                  | see requirements.txt   | Example of gathering local image histogram to compute the global dataset histograms.                                                                            |
```

Comment:
instead of "Federated Statistics for Medical Imaging",
what about just "Federated Statistics for Images"
Since we might use other images there?

Reply:
this works for me